esie_subtheme.theme

```diff
@@ -74,3 +74,22 @@ function esie_subtheme_preprocess_page(&$variables) {
  /* @var Attribute $variables['navbar_top_attributes'] */
   $variables['navbar_top_attributes']->addClass($expand_class);
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter() for node templates.
+ * @param array $suggestions
+ * @param array $variables
+ */
+function esie_subtheme_theme_suggestions_node_edit_form_alter(array &$suggestions, array $variables) {
```

Comment:
A change here:
```PHP
foreach ($roles as $role) {
  $suggestions[] = "node_edit_form__{$role}";
  $suggestions[] = "node_add_form__{$role}";
}
```
Reason: we don't want to hardcode the role and also, since these are suggestions, if the role has no template present then the default will kick in anyway. That makes also adding future templates for other roles way easier.
Thoughts or concerns?